include/verilated.cpp

```diff
@@ -2032,11 +2032,16 @@ bool VlReadMem::get(QData& addrr, std::string& valuer) {
     while (true) {
         int c = std::fgetc(m_fp);
         if (VL_UNLIKELY(c == EOF)) break;
+        const bool c_is_4state_bin
+            = c == '0' || c == '1' || c == 'x' || c == 'X' || c == 'z' || c == 'Z';
```

Comment:
Please use Verilator mixed cap style.

```suggestion
        const bool chIs4StateBin
            = c == '0' || c == '1' || c == 'x' || c == 'X' || c == 'z' || c == 'Z';
```

Reply:
I can make this change, but I was trying to match the existing naming style in the function. Should What are your thoughts?

Reply:
Ah, probably I let it slide when the function was first introduced.  For local variables it's not a big deal, either way.